app/models/spotlight/language.rb

```diff
@@ -0,0 +1,24 @@
+module Spotlight
+  # A language for an exhibit
+  class Language < ActiveRecord::Base
+    belongs_to :exhibit
+    has_many :pages, ->(page) { where('locale = ?', page.locale) }, through: :exhibit
```

Comment:
Can this be `where(locale: page.locale)`, allowing Rails to do its own SQL stringification?